aarch64 does not work with kernels older than 3.7 fails with: "No rule to make target 'crosstool-ng/.build/src/linux-3.2.101/arch/arm64/Makefile"

ctng a6580b8e8b55345a5a342b5bd96e42c83e640ac5 Ubuntu 18.04 host:

```
./ct-ng aarch64-unknown-linux-gnu
./ct-ng menuconfig
```

and pick a kernel older than 3.7 (e.g. 3.2, the oldest).

Then:

```
./ct-ng build
```

fails because apparently `arch/arm64` was added in 3.7 according to `git log arch/arm64`.

The menus should not allow for that.

Error log:

```
[INFO ]  =================================================================
[INFO ]  Installing kernel headers
[EXTRA]    Installing kernel headers
[DEBUG]    ==> Executing:  'make' '-C' '/mnt/sda3/git/crosstool-ng/.build/aarch64-unknown-linux-gnu/src/linux' 'BASH=/mnt/sda3/git/crosstool-ng/.build/tools/bin/bash' 'HOSTCC=x86_64-build_pc-linux-gnu-gcc' 'CROSS_COMPILE=aarch64-unknown-linux-gnu-' 'O=/mnt/sda3/git/crosstool-ng/.build/aarch64-unknown-linux-gnu/build/build-kernel-headers' 'ARCH=arm64' 'INSTALL_HDR_PATH=/mnt/hd/git/crosstool-ng/.build/install/aarch64-unknown-linux-gnu/aarch64-unknown-linux-gnu/sysroot/usr' 'V=0' 'headers_install'
[ALL  ]    make[1]: Entering directory '/mnt/sda3/git/crosstool-ng/.build/src/linux-3.2.101'
[ALL  ]    /mnt/sda3/git/crosstool-ng/.build/src/linux-3.2.101/Makefile:567: /mnt/sda3/git/crosstool-ng/.build/src/linux-3.2.101/arch/arm64/Makefile: No such file or directory
[ERROR]    make[2]: *** No rule to make target '/mnt/sda3/git/crosstool-ng/.build/src/linux-3.2.101/arch/arm64/Makefile'.  Stop.
[ALL  ]    Makefile:130: recipe for target 'sub-make' failed
[ERROR]    make[1]: *** [sub-make] Error 2
[ALL  ]    make[1]: Leaving directory '/mnt/sda3/git/crosstool-ng/.build/src/linux-3.2.101'
[ERROR]  
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Installing kernel headers'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: CT_DoExecLog[scripts/functions@374]
[ERROR]  >>        called from: do_kernel_headers[scripts/build/kernel/linux.sh@63]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@679]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      https://crosstool-ng.github.io/docs/known-issues/
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]  
[ERROR]  (elapsed: 2:26.57)
```

--- a/config/arch/arm.in
+++ b/config/arch/arm.in
@@ -15,6 +15,7 @@
 ## select ARCH_SUPPORTS_WITH_FLOAT if ARCH_32
 ## select ARCH_SUPPORTS_WITH_FPU if ARCH_32
 ## select ARCH_SUPPORTS_SOFTFP if ARCH_32
+## select LINUX_REQUIRE_3_7_or_later if ARCH_64 && KERNEL_LINUX
 
 ## help The ARM architecture, as defined by:
 ## help     http://www.arm.com/

--- a/packages/linux/package.desc
+++ b/packages/linux/package.desc
@@ -1,6 +1,6 @@
 repository='git git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git'
 mirrors='$(CT_Mirrors kernel.org linux ${CT_LINUX_VERSION})'
 relevantpattern='2.*.*|. *.*|.'
-milestones='3.2'
+milestones='3.2 3.7'
 archive_formats='.tar.xz .tar.gz'
 signature_format='unpacked/.sign'

